server/models/services/TestsService.js

```diff
@@ -0,0 +1,27 @@
+const ats = require('../../resources/ats.json');
```

Comment:
Oh thanks for catching this. I must have made a mistake when handling the merge conflicts. Will fix this before merging!

Reply:
> This seems to be based on

By "this", do you mean `server/models/services/TestsService.js` or `server/resources/ats.json`?

Reply:
I meant the `server/models/services/TestsService.js` file.

But I have had a past question on why `ats.json` is being used rather than referencing the db that I don't think has ever been answered (but that has nothing to do with this PR)

Reply:
I believe this is addressed in [de0148d](https://github.com/w3c/aria-at-app/pull/879/commits/de0148d06c7e4dd0d2b71f2128ddbbba8246e61a).

Reply:
> I believe this is addressed in [de0148d](https://github.com/w3c/aria-at-app/pull/879/commits/de0148d06c7e4dd0d2b71f2128ddbbba8246e61a).

Yep, this commit does address the feedback, thanks!

> I left `testsResolver` in place as it makes sense to me to keep a layer between the server and the client. We don't do anything with it currently but authentication, etc may be useful one day. I tried to only use the `getTests` directly in places where the server is retrieving tests from itself.

Understood, that seems fine to me. Thanks for explaining that.